### PR TITLE
Adding interactionStatistic model for like/dislike

### PR DESCRIPTION
**Pull request by ekoi**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -7,3 +7,6 @@ python-dateutil==2.8.2
 lxml==5.1.0
 elementpath==4.4.0
 lorem-text==2.1
+jmespath==1.0.1
```
Comment (kerim1) [inline requirements.txt:10]: This package is not used? So can be removed from the requirements I guess?

File: requirements.txt [hunk 2]
```diff
@@ -7,3 +7,6 @@ python-dateutil==2.8.2
...
+rdflib == 7.0.0
+StrEnum == 0.4.15
```
Comment (kerim1) [inline requirements.txt:12]: Should already be in Python 3.11 which we are using. So can also be removed I guess?

**Review by kerim1 — CHANGES_REQUESTED** [review]
Just some tiny comments

File: requirements.txt (hunk 1 above)
Comment (ekoi) [reply requirements.txt:10]: I use jmeshpath in app.py

Comment (kerim1) [reply requirements.txt:10]: It is not in this pull request?

**Pull request merged**